attribution-graphs-frontend/attribution_graph/init-cg-subgraph.js

```
@@ -0,0 +1,634 @@
+        selForceNodes.forEach((d, i) => {
+          d.fx = d.x = nums[i*2 + 0]/1000*c.width
+          d.fy = d.y = nums[i*2 + 1]/1000*c.height
+        })
+
+        nodeSel.translate(d => [d.x, d.y])
+        styleNodes()
+        renderEdges()
+
+        visState.og_sg_pos = visState.sg_pos
+        delete visState.sg_pos
+      }
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)